static/app/views/performance/newTraceDetails/traceTree.tsx

```diff
@@ -1069,13 +1082,18 @@ export class TraceTreeNode<T extends TraceTree.NodeValue> {
     }
 
     if (isTransactionNode(this)) {
-      this.errors = this.value.errors;
-      this.performance_issues = this.value.performance_issues;
+      for (const error of this.value.errors) {
```

Comment:
I think you could just new Set(this.value.errors) here and on L189, the API might already be reporting non duplicates here, hence why I was just storing the reference

Reply:
Agree